spec/features/edit_training_details_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+
+feature "edit training details" do
```

Comment:
Can we nest this feature spec into `spec/features/trainees`?

Reply:
done